function/go.mod

```diff
@@ -1,41 +1,59 @@
 module github.com/kyma-incubator/hydroform/function
 
-go 1.15
+go 1.18
 
 require (
-	github.com/Microsoft/go-winio v0.4.16-0.20201130162521-d1ffc52c7331 // indirect
-	github.com/containerd/containerd v1.4.3 // indirect
 	github.com/docker/cli v20.10.3+incompatible
-	github.com/docker/distribution v2.7.1+incompatible // indirect
 	github.com/docker/docker v20.10.3+incompatible
 	github.com/docker/go-connections v0.4.0
-	github.com/docker/go-units v0.4.0 // indirect
 	github.com/golang/mock v1.4.4
+	github.com/invopop/jsonschema v0.2.0
+	github.com/moby/moby v20.10.3+incompatible
+	github.com/onsi/gomega v1.10.1
+	github.com/opencontainers/image-spec v1.0.1
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.6.1
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
+	k8s.io/api v0.18.9
+	k8s.io/apimachinery v0.18.9
+	k8s.io/client-go v0.18.9
+)
+
+require (
```

Comment:
doubled `require`?

Reply:
Yes. It's caused by the `go mod tidy` command, probably to split true requires and indirect requires

Reply:
Makes sense